src/utcToZonedTime/test.js

```diff
@@ -1,5 +1,5 @@
 import assert from 'power-assert'
-import format from 'date-fns/format'
+import format from 'date-fns/format.js'
```

Comment:
The move to `date-fns-@3` required this to change to a named import rather than default.

```js
import { format } from 'date-fns/format.js'
```